exercises/practice/list-ops/list_ops_test.rb

```diff
@@ -44,9 +44,9 @@ def test_concat_normal
 
   def test_concat_gigantic
     skip
-    input1 = (1..1_000_000).to_a
-    input2 = (1_000_001..2_000_000).to_a
-    assert_equal (1..2_000_000).to_a, ListOps.concatter(input1, input2)
+    input_1 = (1..1_000_000).to_a
+    input_2 = (1_000_001..2_000_000).to_a
+    assert_equal (1..2_000_000).to_a, ListOps.concatter(input_1, input_2)
```

Comment:
See importance of position vs importance  of uniqueness communication in prior comment.

Since these are (now or later) generated, the maintainer should make this distinction, if this style is enforced, by indicating in the source that this is purposefully done, via a directive comment.  See https://stackoverflow.com/a/26350879/485864 where I talk about inline directive.